adapter/internal/oasparser/envoyconf/constants.go

```diff
@@ -122,3 +125,10 @@ const (
 	// Azure tracer's name
 	TracerTypeAzure = "azure"
 )
+
+// Constants relevant to the rate-limit service
+const (
+	// ALL denotes API level rate limiting
```

Comment:
A small question. Should it be something like 
```suggestion
	// Constants for API rate limiting
```
or can the be a constant as ALL?